### PR TITLE
fix: a quick fix on error message

### PR DESCRIPTION
**Pull request by patrickzheng200**

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — DISMISSED** [review]
LGTM

File: cmd/notation/sign.go [hunk 1]
```diff
@@ -112,8 +112,8 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 	_, err = notation.Sign(ctx, signer, sigRepo, opts)
 	if err != nil {
 		var errorPushSignatureFailed notation.ErrorPushSignatureFailed
-		if errors.As(err, &errorPushSignatureFailed) {
-			return fmt.Errorf("%v. Target registry does not seem to support OCI artifact manifest. Try the flag `--signature-manifest image` to store signatures using OCI image manifest for backwards compatibility", err)
+		if errors.As(err, &errorPushSignatureFailed) && !ociImageManifest {
+			return fmt.Errorf("%v. One possible reason: target registry does not support OCI artifact manifest. Try the flag `--signature-manifest image` to store signatures using OCI image manifest for backwards compatibility", err)
```
Comment (priteshbandi) [inline cmd/notation/sign.go:116]:
```suggestion
			return fmt.Errorf("%v. Possible reason: target registry does not support OCI artifact manifest. Try using flag `--signature-manifest image` to store signatures using OCI image manifest for backwards compatibility", err)
```

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Comment by patrickzheng200** [comment]
Closing this PR as the error message is outdated again due to [PR 573](https://github.com/notaryproject/notation/pull/573).

**Pull request closed without merging**